spec/definitions.yaml

```diff
@@ -318,6 +327,37 @@ FeatureFlagStatuses:
       type: array
       items:
         $ref: '#/definitions/FeatureFlagStatus'
+FeatureFlagStatusForQueriedEnvironment:
+  type: object
+  properties:
+    name:
+      type: string
+      description: |
+        | Name     | Description |
+        | --------:| ----------- |
+        | new      | the feature flag was created within the last 7 days, and has not been requested yet |
+        | active   | the feature flag was requested by your servers or clients within the last 7 days |
+        | inactive | the feature flag was created more than 7 days ago, and hasn't been requested by your servers or clients within the past 7 days |
+        | launched | one variation of the feature flag has been rolled out to all your users for at least 7 days |
+      enum:
+        - new
+        - active
+        - inactive
+        - launched
+    lastRequested:
+      type: string
+      example: "2016-08-16T21:10:11.886Z"
+    default:
+      type: object
+FeatureFlagStatusAcrossEnvironments:
+  type: object
+  properties:
+    _links:
+      $ref: '#/definitions/Links'
+    environments:
+      type: object
+      environmentKey:
```

Comment:
I think this needs to be specified something like this:

```
properties:
  _links:
    $ref: '#/definitions/Links'
  key:
    type: string
  environments:
    type: object
    additionalProperties:
      $ref: '#/definitions/FeatureFlagStatusForQueriedEnvironment'
```

In order to define the type of the objects within the dictionary.